frontend/templates/product/sections/CrossSellSection/index.tsx

```diff
@@ -181,7 +181,7 @@ export function CrossSellSection({
                }}
                size="lg"
```

Comment:
I might also do 
```suggestion
               size="lg"
               textTransform="capitalize"
```

for these

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform

Reply:
Drawback to this:

> Note: Authors should not expect capitalize to follow language-specific title casing conventions (such as skipping articles in English).